src/data/fintype.lean

```diff
@@ -682,3 +682,26 @@ lemma bijective_bij_inv (f_bij : bijective f) : bijective (bij_inv f_bij) :=
 end bijection_inverse
 
 end fintype
+
+class infinite (α : Type*) : Prop :=
```

Comment:
maybe make it more constructive/explicit/easier-to-use by saying that for every finset there is a member not in it?

Reply:
I proved that lemma. I quite like this as the definition, because it means that transferring the lemmas from `fintype` is just a case of modus tollens.